### PR TITLE
Qna search

### PR DESCRIPTION
**Pull request by Taylor-Sheets3**
Implemented: 

1. Qna search bar functionality
      1.1 Searches return results based on question body text
      1.2 Adding questions while searching expands the list of displayed questions while applying the search filter
2. Photos can be submitted with answers
3. Answers display photos after answer text and before answer details

### CODE REVIEW

File: client/src/components/questionsAnswers/D-QuestionsCards.jsx [hunk 1]
```diff
@@ -48,18 +55,20 @@ function QuestionsCards(props) {
...
+  function handleModalSubmit(text, nickname, userEmail, photos) {
     const body = {
       body: text,
       name: nickname,
       email: userEmail,
-      photos: ['placeholder'],
+      photos,
     };
     axios.post(`/qa/questions/${question_id}/answers`, body)
+      .then((response) => console.log(response))
```
Comment (KevZhang11) [inline client/src/components/questionsAnswers/D-QuestionsCards.jsx:66]: don't forget to remove the console logs later on =)

Comment (Taylor-Sheets3) [reply client/src/components/questionsAnswers/D-QuestionsCards.jsx]: Thanks Kevin! Change is pushed!

File: client/src/components/questionsAnswers/D-QuestionsCards.jsx [hunk 2]
```diff
@@ -108,19 +117,21 @@ function QuestionsCards(props) {
...
+              setNoAs(answers.length);
+            } else {
+              setNoAs(2);
+            }
+            setMoreAs(!moreAs);
+          }}
+        >
+          {moreAs ? 'Collapse answers' : 'Load more answers'}
+        </button>
+      ) : (<br />)}
```
Comment (KevZhang11) [inline client/src/components/questionsAnswers/D-QuestionsCards.jsx:133]: I could be wrong, but it looks like the paren and bracket are not lining up

Comment (Taylor-Sheets3) [reply client/src/components/questionsAnswers/D-QuestionsCards.jsx:133]: The opening bracket corresponding to the closing bracket you see there on line 134 is all the way up on line 119.  It lines up :)

**Review by KevZhang11 — CHANGES_REQUESTED** [review]
Good work! just 2 minor comments i left. Feel free to let me know what you think and i'll merge it after

File: client/src/components/questionsAnswers/D-QuestionsCards.jsx (hunk 2 above)
Comment (KevZhang11) [reply client/src/components/questionsAnswers/D-QuestionsCards.jsx:133]: got cha

**Review by KevZhang11 — APPROVED** [review]
Looks good!

**Pull request merged**